lib/std/os/windows/ws2_32.zig

```diff
@@ -7,10 +7,879 @@ usingnamespace @import("bits.zig");
 
 pub const SOCKET = *opaque {};
 pub const INVALID_SOCKET = @intToPtr(SOCKET, ~@as(usize, 0));
-pub const SOCKET_ERROR = -1;
 
```

Comment:
maybe this file should have a comment pointing out that some of the constants are from https://github.com/marlersoft/zigwin32 ?

This would help anyone modifying this file in the future

Reply:
Sure, though the comment may better belong in an another file - if it made sense, I would've preferred having the entirety of the bindings in the standard library, but it unfortunately is ~1.6MiB of code 😅.

As constant and syscall definitions will eventually be pulled from the auto-generated bindings from other modules liker kernel32, maybe a good place to put the comment might just be in `std.os.windows`?